discord/guild.py

```diff
@@ -267,6 +267,10 @@ class Guild(Hashable):
         Indicates if the guild has widget enabled.
 
         .. versionadded:: 2.0
+    max_stage_video_channel_users: Optional[:class:`int`]
```

Comment:
This name to me feels like it's an instance where we should deviate from discord.

Reply:
What about `max_stage_video_users`?

Reply:
Changed it to `max_stage_video_users`